src/helpers.spec.ts

```diff
@@ -0,0 +1,167 @@
+// tslint:disable:no-implicit-dependencies
+import 'mocha';
+import { assert } from 'chai';
+import { getTypeAndConversation, IncomingEventType } from './helpers';
+
+describe('getTypeAndConversation()', () => {
+  describe('event types', () => {
+    // Arrange
+    const conversationId = 'CONVERSATION_ID';
+    const dummyEventBody = {
+      event: {
+        type: 'app_home_opened',
+        channel: conversationId,
+        event_ts: 'EVENT_TS',
```

Comment:
i don't think `type` or `event_ts` are technically needed for the tests to succeed. right?